examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-cylinder-bed.prm

```diff
@@ -204,8 +208,8 @@ subsection lagrangian physical properties
   end
   set young modulus wall           = 1e7
   set poisson ratio wall           = 0.25
-  set restitution coefficient wall = 0.33
-  set friction coefficient wall    = 0.2
+  set restitution coefficient wall = 0.97
+  set friction coefficient wall    = 0.4
   set rolling friction wall        = 0.3
```

Comment:
Rolling friction should be something like 0.01, right?

Reply:
It can be much higher than 0.01, for example Toni's case had >0.2